portfolio/src/main/webapp/index.html

```diff
@@ -166,8 +166,45 @@ <h2>Connect with me</h2>
   <a class="nav-link" href="#">Return to Top</a>
 </div>
 <div id="comment-section">
+  <p>Have a feedback ? Please leave a comment
+  <button class="btn btn-outline-dark btn-sm" type="button" data-toggle="collapse"
+      data-target="#comment-form" aria-expanded="false" aria-controls="comment-form">
+    Add Comment
+  </button>
+
+  <form action="/data" method="POST" class="collapse" id="comment-form">
+    <p>Please complete the Comment form
+    <table>
+      <tr class="form-row">
+        <td class="form-group col-md-6">
+          <label for="first-name">First Name</label>
+          <input type="text" class="form-control" placeholder="First Name" id="first-name" name="first-name">
+        <td class="form-group col-md-6">
+          <label for="last-name">Last Name</label>
+          <input type="text" class="form-control" placeholder="Last Name" id="last-name" name="last-name">
+      <tr class="form-row">
+        <td>
+          <label for="comment">Comment</label>
+          <input type="text" class="form-control" placeholder="Write your comment" id="comment" name="comment">
+        <td>
+          <button type="submit" class="btn btn-outline-dark btn-sm">Submit</button>
+    </table>
+  </form>
```

Comment:
I'm not sure the `<table>` is appropriate here since this is a form and the layout is more of a grid than a strict table. Probably fine to just use `<div>` elements.

The following is just a starting point. I'm not sure if `col-md-6` is even necessary.

```html
  <form action="/data" method="POST" class="collapse" id="comment-form">
    <p>Please complete the Comment form
    <div class="form-row">
      <div class="form-group col-md-6">
        <label for="first-name">First Name</label>
        <input type="text" class="form-control" placeholder="First Name" id="first-name" name="first-name">
      </div>
      <div class="form-group col-md-6">
        <label for="last-name">Last Name</label>
        <input type="text" class="form-control" placeholder="Last Name" id="last-name" name="last-name">
      </div>
    </div>
    <div class="form-group">
      <label for="comment">Comment</label>
      <input type="text" class="form-control" placeholder="Write your comment" id="comment" name="comment">
    </div>
    <button type="submit" class="btn btn-outline-dark btn-sm">Submit</button>
  </form>
```

Reply:
I was not done with UI cleanup and used table temporarily. Thanks for the suggestion.

Reply:
I am using col-md-6 to have the two fields equally split in the screen.